baseline.xml

```diff
@@ -3,17 +3,4 @@
   <file src="src/ComposerRequireChecker/Cli/CheckCommand.php">
     <MixedArgumentTypeCoercion occurrences="2"/>
```

Comment:
Oh, last one left? Can we completely get rid of the baseline, perhaps?

Reply:
Currently I don’t know how to process all the ongoing generator stuff ^^